packages/react-experiments/src/components/Slider/Slider.types.ts

```diff
@@ -151,92 +181,112 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
    * If marks is an array, it should contain objects with keys of `value` (denote the where to place the label on the
    * slider if specified) and `label` (what to label to display at specified position).
    * @defaultvalue false
+   * @deprecated
    */
   marks?: boolean | ISliderMarks[];
 
   /**
    *  Optional flag to decide that thumb will snap to closest value while moving the slider.
    *  @defaultvalue false
+   * @deprecated
    */
   snapToStep?: boolean;
 }
 
 /**
+ * @deprecated
  * {@docCategory Slider}
  */
 export type ISliderStyleProps = Required<Pick<ISliderProps, 'theme'>> &
   Pick<ISliderProps, 'className' | 'disabled' | 'vertical'> & {
+    /** @deprecated */
```

Comment:
I don't think we need to mark props inside a deprecated interface as deprecated individually (though I guess it could give better tooling hints)

Reply:
Yeah, that was my reasoning.